src/core/src/lib.rs

```diff
@@ -145,7 +148,9 @@ pub enum Primitive {
     /// Every three consecutive vertices represent a single triangle. For example, with `[a, b, c,
     /// d]`, `a`, `b`, and `c` form a triangle, and `b`, `c`, and `d` form a triangle.
     TriangleStrip,
-    //Quad,
+    /// Quad,
+    /// 4 point quad patch
+    QuadList
```

Comment:
There is a problem here. The implementation you got only supports the quad list for tessellation, while the name doesn't bear this limitation, so some users may be confused trying to render non-tessellated quads.